core/src/main/java/de/danoeh/antennapod/core/util/FileNameGenerator.java

```diff
@@ -34,7 +35,11 @@ public static String generateFileName(String string) {
 		}
 		String filename = buf.toString().trim();
 		if(TextUtils.isEmpty(filename)) {
-			return RandomStringUtils.randomAlphanumeric(8);
+			return new RandomStringGenerator.Builder()
+					.withinRange('0', 'z')
```

Comment:
Between ASCII `0` (0x30) and ASCII `z` (0x7a), there are non alphanumeric characters like `>=[\;`.

I guess instead of 0-z, you should do 0-9,a-z